### DISCUSSION TITLE
Problem while updateing nuget package

### DISCUSSION THREAD

**Discussion by AdamKlob** [SixLabors/Fonts#368] [General]
I'm trying to update from 1.0.0-beta15 to latest version, but I ger an error that my (C# Windows Forms) project target „.NETFramework,Version=v4.8.1” is not supported by the package. How can I update?

**Comment by dlemstra** [comment]
Starting with version `2.0.0` support for .NET framework has been removed. So this means you can no longer use the latest version without upgrading your project.

**Reply by JimBobSquarePants** [reply]
v1.0.0 will cover most needs but if possible I would definitely upgrade your target framework as there are a few fixed issues only present in v2.0.0 and above.

https://github.com/SixLabors/Fonts/releases
